src/canvas.class.js

```diff
@@ -1349,10 +1353,25 @@
     },
 
     setViewportTransform: function (vpt) {
```

Comment:
Since you're working on this function it'd probably be good to add some documentation.

Reply:
994ef12